### PR TITLE
initial implement with test

### PR DESCRIPTION
**Pull request by meomap**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,7 @@
-# logruswindow
\ No newline at end of file
+# logruswindow
```
Comment (bclermont) [inline README.md]: window**s**

https://en.wikipedia.org/wiki/Microsoft_Windows

File: log.go [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+		return nil, errors.Wrapf(err, "eventlog.Open source=%s", source)
+	}
+	return &EventHook{
+		Log:    l,
+		source: source,
+		levels: levels,
+	}, nil
+}
+
+// Fire extracts logrus entry and send to window event log
```
Comment (hvnsweeting) [inline log.go]: sends to

File: log.go [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+	}
+	return errors.Errorf("Unknown logrus level %s", entry.Level.String())
+}
+
+// Levels returns current available logging levels.
+func (hook *EventHook) Levels() []logrus.Level {
+	return hook.levels
+}
+
+// Close event log & remove registry
```
Comment (hvnsweeting) [inline log.go]: removes

Comment (meomap) [inline log.go]: @hvn thanks, fixed all typos

File: README.md [hunk 4]
```diff
@@ -1 +1,28 @@
-# logruswindow
```
Comment (favadi) [inline README.md:1]: please renames to logruswindows, window means: https://upload.wikimedia.org/wikipedia/commons/2/20/Window_Porto_Covo_August_2013-2.jpg.

Comment (favadi) [inline README.md:1]: Sorry, It is already logruswindows, ignore it.

**Comment by bclermont** [comment]
👍

**Comment by favadi** [comment]
LGTM! I created #2 for travis integration. Can you enable it?

File: log.go [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+}
+
+// Fire extracts logrus entry and sends to window event log
+func (hook *EventHook) Fire(entry *logrus.Entry) error {
+	msg, err := entry.String()
+	var eventID uint32 = 1
+	entryEventID, ok := entry.Data["event_id"]
+	if ok {
+		switch id := entryEventID.(type) {
+		case int:
```
Comment (bclermont) [inline log.go]: what if it's `int32`, `int8`, `int64`, `uint`, `uint8` and `uint64`?

Comment (meomap) [inline log.go]: they're all ignored by now, other hook such as sentry logrus support string only, I think we should pass event id as string also, for compatibility

Comment (bclermont) [inline log.go]: you just need to append those types to line 44

`case int, int32, int8, int64, uint, uint8, uint64:`

Comment (meomap) [inline log.go]: done at 22788e

**Pull request merged**